pkg/client/api/openapi.yaml

```diff
@@ -218,6 +218,7 @@ paths:
         required: false
         schema:
           example: c336f57e,476547a8
+          maxItems: 25
```

Comment:
Is `maxItems` the right key here?  
The query param is a string of a list of ids: `?customerIDs=id1,id2,...`

Reply:
Actually, I should tweak the schema here:
```
schema:
  type: array
  items:
    type: string
...
```
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml#L89

Reply:
There's a difference between these three. We want the first.

- `?customerIDs=c1,c2,c3` (`type: string`)
- `?customerIDs=c1&customerIDs=c2` (`type: array`)
- `?customerIDs=[c1,c2,c3]` 
- `?customerIDs[]=c1,c2,c3`

Reply:
Oof, my bad, thanks for the clarification!

And would the `maxItems: 25` be ok for the first one? I can't find anything in the swagger spec to confirm this.

Reply:
I think `maxItems: <number>` is only valid on `type: array`, but this parameter is `type: string`.

Reply:
Hmm `make client` worked; do you recommend just specifying this limit in the description?

Reply:
Yea. We probably want to reject requests with 26 customerID's, but that seems annoying for callers.

Reply:
Updated the docs here.